Makefile

```diff
@@ -1,11 +1,31 @@
 
 IMAGE = sstarcher/newrelic-operator:test
 
-.PHONY: test
+.PHONY: all build
 
-test:
+all: e2e
+
+dep:
+	dep ensure
+
+build: dep
+ifdef NEW_RELIC_APIKEY
 	git checkout deploy/operator.yaml
 	@sed -i '' s/REPLACE_ME_NEW_RELIC_APIKEY/${NEW_RELIC_APIKEY}/ deploy/operator.yaml
+	@sed -i '' s/REPLACE_NAMESPACE/default/ deploy/role_binding.yaml
```

Comment:
should we also checkout this file to reset it?

Reply:
Good point. Done